src/js/components/Pagination/Pagination.js

```diff
@@ -199,9 +201,10 @@ const Pagination = forwardRef(
       ...navProps[control],
     }));
 
-    return (
+    let content = (
       <StyledPaginationContainer
         flex={false}
+        wrap
```

Comment:
Should `wrap` only be added when `stepSelector` is defined?

Reply:
took out didnt need